README.md

```diff
@@ -107,6 +107,7 @@ Following are the list of some popular blogging platforms and their RSS feed url
 | [Anchor.fm Podcasts](https://anchor.fm/) | `https://anchor.fm/s/podcastId/podcast/rss` | You can get the rss feed url of a podcast by following [these](https://help.anchor.fm/hc/en-us/articles/360027712351-Locating-your-Anchor-RSS-feed) instructions | https://anchor.fm/s/1e784a38/podcast/rss |
 | [Hashnode](https://hashnode.com/) | `https://@username.hashnode.dev/rss.xml` | Replace @username with your Hashnode username | https://polilluminato.hashnode.dev/rss.xml |
 | [Google Podcasts](https://podcasts.google.com/) | `https://podcasts.google.com/feed/channelId` | Replace `channelId` with your Google podcast channel Id | https://podcasts.google.com/feed/aHR0cHM6Ly9mZWVkcy5zb3VuZGNsb3VkLmNvbS91c2Vycy9zb3VuZGNsb3VkOnVzZXJzOjYyOTIxMTkwL3NvdW5kcy5yc3M= |
+| [Reddit](https://www.reddit.com/) | `https://www.reddit.com/r/topic/` | Replace `topic` (example: r/gadgets or r/bayarea) with Reddit topic that interest you or localized to you.| https://www.reddit.com/r/bayarea/ |
```

Comment:
Feed URL: `https://any-reddit.com-url.rss`
Comments: `You can create an RSS feed by adding ".rss" to the end of an existing Reddit URL`
Example: `https://www.reddit.com/r/bayarea.rss`

Please make these changes